### PR TITLE
Resolve review comments

### PR DESCRIPTION
**Pull request by WhiteMinds**
Refs: https://nervos-official-website-git-preview-magickbase.vercel.app/

### CODE REVIEW

File: src/pages/community/index.page.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import styles from './index.module.scss'
 
 import { CommunityHubFloatIconGroup } from './icons'
 
-const title = <div>Nervos Community Hub.</div>
+const title = <div>Nervos community hub.</div>
```
Comment (Keith-CY) [inline src/pages/community/index.page.tsx:15]: The title is capitalized in https://docs.google.com/document/d/1abU8b1RSSDIXN3IquKbdT7ukxLCoIyaZg8F32-XtJow/edit

File: src/pages/wallets/index.page.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ import {
   ImTokenIcon,
 } from './icons'
 
-const title = <div>Download a Wallet and Start Exploring the Nervos Ecosystem .</div>
+const title = <div>Download a wallet.</div>
```
Comment (Keith-CY) [inline src/pages/wallets/index.page.tsx:23]: The title is expected to be `Download a Wallet and Start Exploring the Nervos Ecosystem` in https://docs.google.com/document/d/1abU8b1RSSDIXN3IquKbdT7ukxLCoIyaZg8F32-XtJow/edit

File: src/pages/developers/index.page.tsx [hunk 3]
```diff
@@ -91,11 +91,11 @@ const resourceData = {
...
-    // {
-    //   title: 'CKB Training Course',
-    //   content:
-    //     'Detailed tutorials about creating CKB transactions and details about the structure of the Layer 1 CKB blockchain.',
-    // },
+    {
+      title: 'Nervos Talk Forum',
+      content: 'For the community to participate and discuss developments on Nervos.',
+      link: 'https://talk.nervos.org/',
+    },
```
Comment (Keith-CY) [inline src/pages/developers/index.page.tsx:98]: Only the first 3 links are mentioned in https://docs.google.com/document/d/1abU8b1RSSDIXN3IquKbdT7ukxLCoIyaZg8F32-XtJow/ as developer resources

Comment (WhiteMinds) [reply src/pages/developers/index.page.tsx:98]: This is the designer's comment https://nervos-official-website-git-preview-magickbase.vercel.app/developers#thread-id=Jdgb5

File: src/pages/wallets/index.page.tsx (hunk 2 above)
Comment (WhiteMinds) [reply src/pages/wallets/index.page.tsx:23]: This is the designer's comment https://nervos-official-website-git-preview-magickbase.vercel.app/wallets#thread-id=_pVf5

File: src/pages/community/index.page.tsx (hunk 1 above)
Comment (WhiteMinds) [reply src/pages/community/index.page.tsx:15]: This is the designer's comment https://nervos-official-website-git-preview-magickbase.vercel.app/community#thread-id=3NxPi

**Pull request merged**